ICSharpCode.Decompiler/IL/Transforms/ExpressionTransforms.cs

```diff
@@ -720,6 +723,7 @@ void TransformCatchVariable(TryCatchHandler handler, Block entryPoint)
 					if (inlinedUnboxAny.Type.Equals(handler.Variable.Type)) {
 						context.Step("TransformCatchVariable - remove inlined UnboxAny", inlinedUnboxAny);
 						inlinedUnboxAny.ReplaceWith(inlinedUnboxAny.Argument);
+						(ilOffsets ?? (ilOffsets = new List<Interval>())).AddRange(inlinedUnboxAny.ILRanges);
```

Comment:
Dead store -- `ilOffsets` is never used after this.
Which in turn means the whole `ilOffsets` variable is unnecessary.

Reply:
OK, what should happen to the "lost" sequence point of `inlinedUnboxAny`?

Reply:
If you want to save it in `handler.AddExceptionSpecifierILRange`, you need to do so before the `return;`.